modules/common-workflows/pages/workflow-install-from-raw-image.adoc

```diff
@@ -1,62 +1,61 @@
-[[workflow-inplace-sles-upgrade]]
+[[workflow-install-from-raw-image]]
+= Installing {productname} Using RAW Image
 
-= Installing {productname} using raw image
 
+{productname} offers various image formats, including a RAW format. 
+This guide demonstrates how to set up {productname} Server using the raw image.
 
-{productname} offers various image formats, including a raw format. This guide demonstrates how to set up {productname} Server using the raw image.
 
 
 == Use Case
 
-RAW images are pre-built, ready-to-use representations of a running operating system. Instead of being installed through a traditional installer, they are directly copied to the hard disk of the target host.
-Raw image format is flexible and compatible with a wide range of virtualization and deployment tools. This guide provides essential information about these pre-configured images.
+RAW images are pre-built, ready-to-use representations of a running operating system. 
+Instead of being installed through a traditional installer, they are directly copied to the hard disk of the target host.
+RAW image format is flexible and compatible with a wide range of virtualization and deployment tools. 
+
+This guide provides essential information about these pre-configured images.
+
 
 
 == Outcome 
 
 Successful completion of this workflow results in successful installation of {productname} Server.
 
 
-== Step-by-step Preparation Instructions
 
+== Step-by-step Instructions
 
 [WARNING]
 ====
 This workflow is complex and time-consuming.
 Make sure that adequate testing is done before deploying the procedure in live environment.
 ====
 
-.Procedure: Prerequisites
+.Procedure: Prepare the Prerequisites
 [role=procedure]
 . *Download the Image:* Download the architecture-specific [literal]``.raw.xz`` image of SUSE Manager 5.0 Server for your target block size.
-
-
-. *Make Image Accessible:* Copy the image to a web server (preferred), NFS server, or expand the .raw file to add directly as a disk in a virtual environment (if supported).
-
-
-. *Boot Media:* Provide a bootable ISO ({sles} 15 SPX or {sle-micro} 5.5) to access the "Rescue System." Add netsetup=1 on the kernel command line for networking, or configure IP via ifcfg.
-
+. *Make Image Accessible:* Copy the image to a web server (preferred), NFS server, or expand the [literal]``.raw`` file to add directly as a disk in a virtual environment (if supported).
```

Comment:
We should remove the NFS server from this list - as the instructions are more complicated, and copying it to a webserver is not an onerous requirement.  
We should include the 'dd' instructions if we give the option to direct mount the '.raw' disk - as this doc does not yet contain those steps.

Reply:
@dvosburg - I have removed NFS server from the list according to your instructions, but I need more details about the second part of this comment. What do 'dd' instructions need to look like?

Reply:
If you used NFS as the server type from which to serve the image, use dd to copy it:
For example if the .raw is expanded:
dd if=SUSE-Manager-Server.ppc64le-5.0.0-Raw-ppc-4096-GM.raw of=/dev/sda bs=4M status='progress'
Use dd to copy the compressed raw image to the root disk like this:
xz -d SUSE-Manager-Server.ppc64le-5.0.0-Raw-ppc-4096-GM.raw.xz | dd of=$device bs=4M status='progress'